setup/Microsoft.VisualStudio.NuGet.BuildTools.vsmanproj

```diff
@@ -10,6 +10,7 @@
     <BuildNumber>$(SemanticVersion).$(BuildNumber)</BuildNumber>
     <TargetName>$(MSBuildProjectName)</TargetName>
     <OutputPath>$(VsixPublishDestination)</OutputPath>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
Out of curiosity, do you remember why this is needed?  This file builds NuGet's "package" for the VS installer. I wouldn't have expected it to need changes when changing the SDK version for `dotnet.integration.tests`. I guess moving the build to the VS2022 agent pool might have caused something to change. I'm just surprised.

Reply:
The setup\Microsoft.VisualStudio.NuGet.BuildTools.vsmanproj is targeting net4.0, so [it is not able to build on VS2022](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5598243&view=logs&j=d80b2849-30d4-52dd-74cd-d6536ba98fe0&t=586e46ba-f09c-562a-bcb6-0b16610ca3f5&l=14).
According to [this doc](https://docs.microsoft.com/en-us/dotnet/framework/install/guide-for-developers), Visual Studio 2022 and later versions can't build apps that target .NET Framework 4.0 through .NET Framework 4.5.1.